blogs/ecs-service-connectivity/yelb/deployments/platformdeployment/AWS/ECS/deploy_yelb.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -eo pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+source ${DIR}/.settings
+
+if [ -z $AWS_ACCOUNT_ID ]; then
+    echo "AWS_ACCOUNT_ID environment variable is not set."
+    exit 1
+fi
+
+if [ -z $AWS_DEFAULT_REGION ]; then
+    echo "AWS_DEFAULT_REGION environment variable is not set."
+    exit 1
+fi
+
+if [ -z $ENVOY_IMAGE ]; then
+    echo "ENVOY_IMAGE environment variable is not set to App Mesh Envoy, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html"
+    exit 1
+fi
+
+if [ -z $VPC ]; then
+    echo "VPC environment variable is not set. Before proceeding setup a VPC with 2 public subnets and continue."
+    exit 1
+fi
+
+if [ -z $PUBLIC_SUBNET_1 ]; then
+    echo "PUBLIC_SUBNET_1 environment variable is not set. Before proceeding setup a VPC with 2 public subnets and continue."
+    exit 1
+fi
+
+
+if [ -z $PUBLIC_SUBNET_2 ]; then
+    echo "PUBLIC_SUBNET_2 environment variable is not set. Before proceeding setup a VPC with 2 public subnets and continue."
+    exit 1
+fi
+
+
+PROJECT_NAME=yelb
+ECR_IMAGE_PREFIX=${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${PROJECT_NAME}
+YELB_UI_SRC="${DIR}/../../../../yelb-ui"
+YELB_UI_IMAGE="${ECR_IMAGE_PREFIX}/yelb-ui:$(git log -1 --format=%h ${YELB_UI_SRC})"
+YELB_APP_SRC="${DIR}/../../../../yelb-appserver"
+YELB_APP_IMAGE="${ECR_IMAGE_PREFIX}/yelb-appserver:$(git log -1 --format=%h ${YELB_APP_SRC})"
+AWS_CLI_VERSION=$(aws --version 2>&1 | cut -d/ -f2 | cut -d. -f1)
+
+ecr_login() {
+    if [ $AWS_CLI_VERSION -gt 1 ]; then
+        aws ecr get-login-password --region ${AWS_DEFAULT_REGION} | \
+            docker login --username AWS --password-stdin ${ECR_URL}
```

Comment:
`ECS_URL` is not defined here.

Reply:
good catch. Added it https://github.com/aws/aws-app-mesh-examples/pull/325/commits/c1c9e31a33d8cd1326e5c1f42db289f594e9f19b